src/lib/useKeyfileGenerator.ts

```diff
@@ -18,20 +18,35 @@ import {
 } from './keys';
```

Comment:
This is the legacy implementation, and is no longer consumed by the application. Keeping it for comparison purposes during the review. We can remove it as a final commit to this PR, or in a follow up.